web/concrete/src/Console/Command/ConfigCommand.php

```diff
@@ -0,0 +1,172 @@
+            }
+            switch ($input->getArgument('operation')) {
+                case self::OPERATION_GET:
+                    $output->writeln(self::serialize(\Config::get($key)));
+                    break;
+                case self::OPERATION_SET:
+                    $value = $input->getArgument('value');
+                    if (!isset($value)) {
+                        throw new Exception('Missing new configuration value');
+                    }
+                    \Config::save($key, self::unserialize($value));
+                    break;
+                default:
+                    throw new Exception('Invalid operation specified. Allowed operations: '.implode(', ', self::getAllowedOperations()));
+            }
+        } catch (Exception $x) {
+            $output->writeln('<error>'.$x->getMessage().'</error>');
+            $rc = 1;
+        }
+
+        return $rc;
+    }
+
+    /**
+     * @return string[]
+     */
+    protected static function getAllowedOperations()
+    {
+        return array(
+            self::OPERATION_GET,
+            self::OPERATION_SET,
+        );
+    }
+
+    /**
+     * @param string $environment
+     *
+     * @return bool
+     */
+    protected static function isValidConfigurationKeyName($key)
+    {
+        return (is_string($key) && preg_match('/^[a-z]\w*(\.\w+)+$/i', $key)) ? true : false;
+    }
+
+    /**
+     * @param mixed $value
+     *
+     * @return string
+     *
+     * @throws Exception
+     */
+    protected static function serialize($value)
+    {
+        $jsonOptions = 0;
+        if (defined('JSON_UNESCAPED_SLASHES')) {
+            $jsonOptions |= JSON_UNESCAPED_SLASHES;
+        }
+        $type = gettype($value);
+        $result = null;
+        switch ($type) {
+            case 'boolean':
+                $result = $value ? 'true' : 'false';
+                break;
+            case 'NULL':
+                $result = 'null';
+                break;
+            case 'integer':
+            case 'double':
+                $result = (string) $value;
+                break;
+            case 'string':
+                $enquote = false;
+                switch ($value) {
+                    case 'true':
+                    case 'false':
+                    case 'null':
+                        $enquote = true;
+                    default:
+                        if (preg_match('/^-?\d+(\.\d*)?$/', $value)) {
+                            $enquote = true;
+                        }
+                        break;
+                }
+                $result = $enquote ? "\"$value\"" : $value;
+                break;
+        }
+        if (!isset($result)) {
+            throw new Exception("Unable to represent variable of type '$type'");
+        }
+
+        return $result;
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return mixed
+     *
+     * @throws Exception
+     */
+    protected static function unserialize($value)
```

Comment:
Here's an [example of what it'd be if we used `json_decode()`](http://3v4l.org/QqXtJabrum), what do you think @mlocati